templates/PriceHistory/Index/_breadcrumb.html.twig

```diff
@@ -0,0 +1,9 @@
+{% import '@SyliusAdmin/Macro/breadcrumb.html.twig' as breadcrumb %}
+
+{{ breadcrumb.crumble([
+    { label: 'sylius.ui.administration'|trans, url: path('sylius_admin_dashboard') },
+    { label: 'sylius.ui.product'|trans , url: path('sylius_admin_product_show', {'id': app.request.get('productId') })},
+    { label: 'sylius.ui.variant'|trans , url: path('sylius_admin_product_variant_update', {'id': app.request.get('variantId'), 'productId': app.request.get('productId') })},
+    { label: 'sylius_price_history_plugin.ui.price_history'|trans },
+    ])
+}}
```

Comment:
To be consistent with other breadcrumbs, it should probably look more like: `Administration > Products > Product name > Variants > Variant name > Channel pricings > Channel code > History` 🤔